timed/projects/filters.py

```diff
@@ -69,16 +72,24 @@ class Meta:
         """Meta information for the project filter set."""
 
         model = models.Project
-        fields = ["archived", "customer", "billing_type", "cost_center", "reference"]
+        fields = (
+            "archived",
+            "customer",
+            "billing_type",
+            "cost_center",
+            "reference",
+        )
 
 
 class MyMostFrequentTaskFilter(Filter):
     """Filter most frequently used tasks.
 
-    TODO:
+    Todo:
+    ----
     From an api and framework standpoint instead of an additional filter it
     would be more desirable to assign an ordering field frecency and to
     limit by use paging.  This is way harder to implement therefore on hold.
+
     """
```

Comment:
I think TODO or similar comments that dont *describe* the code should be comments, not docstrings

```suggestion
    """
    # TODO: From an api and framework standpoint instead of an additional filter it
    # would be more desirable to assign an ordering field frecency and to
    # limit by use paging.  This is way harder to implement therefore on hold.
```